### PR TITLE
qemu-kvm: bump to 20190827 and use gcc-7

### PR DESCRIPTION
**Pull request by alarcher**
Updated remove-vnd.patch

### CODE REVIEW

File: components/openindiana/qemu-kvm/patches/remove-vnd.patch [hunk 1]
```diff
@@ -9,35 +9,133 @@ This reverts commit 31a39c6ea8dfc9021436264d83dc07a4d799ff61.
...
 -XLDFLAGS="${XLDFLAGS} -lz -lm -lc -lvnd"
 +XLDFLAGS="${XLDFLAGS} -lz -lm -lc"
  ./configure \
      --cc=$CC \
      --extra-cflags="${XCFLAGS}" \
-diff --git a/net/vnic.c b/net/vnic.c
-index af60741..d03ba32 100644
---- a/net/vnic.c
-+++ b/net/vnic.c
+diff -ruN illumos-kvm-cmd-41e7c3e192eb39bb5ced7667fdc12ad2b33e646d.orig/build.sh.orig illumos-kvm-cmd-41e7c3e192eb39bb5ced7667fdc12ad2b33e646d.patched/build.sh.orig
```
Comment (pyhalov) [inline components/openindiana/qemu-kvm/patches/remove-vnd.patch]: do we need this sh.orig file?

File: components/openindiana/qemu-kvm/Makefile [hunk 2]
```diff
@@ -18,12 +18,12 @@ include ../../../make-rules/shared-macros.mk
 
 PATH=	/opt/onbld/bin/i386:/usr/bin:/usr/sbin:/sbin:/usr/gnu/bin
 COMPONENT_NAME=		illumos-kvm-cmd
-COMPONENT_VERSION=	c1a197c8e4582c68739ab08f7e3198b2392c9820
-COMPONENT_REVISION=	2
-IPS_COMPONENT_VERSION=	0.0.1.20160303
+COMPONENT_VERSION=	41e7c3e192eb39bb5ced7667fdc12ad2b33e646d
+COMPONENT_REVISION=	3
```
Comment (AndWac) [inline components/openindiana/qemu-kvm/Makefile]: I wonder whether it we can drop COMPONENT_REVISION here as the package will get completely new version information anyways.

Comment (pyhalov) [reply components/openindiana/qemu-kvm/Makefile]: yes, we can

Comment (Toasterson) [reply components/openindiana/qemu-kvm/Makefile]: IIRC a git revision should break IPS's update behavior. In my research it is explicitly mentioned that the solver tries to work with Semantic versions in the main. When we use incorporations, then it won't matter but without we still need some version number that follows semantic versioning.

Comment (AndWac) [reply components/openindiana/qemu-kvm/Makefile]: The actual FRMI is pkg://openindiana.org/system/qemu/kvm@0.0.1.20160303-2018.0.0.2:20180912T131705Z
The package will get a new Component Version (0.0.1.20190827) and a new Branch Version (either 2020.0.0.3 or 2020.0.0.0 depending on the COMPONENT_REVISION).

File: components/openindiana/qemu-kvm/patches/remove-vnd.patch (hunk 1 above)
Comment (alarcher) [reply components/openindiana/qemu-kvm/patches/remove-vnd.patch]: No sorry I forgot to remove it...

File: components/openindiana/qemu-kvm/Makefile (hunk 2 above)
Comment (alarcher) [reply components/openindiana/qemu-kvm/Makefile]: I did forget to remove this line after bumping to the 2019 tag.

File: components/openindiana/qemu-kvm/patches/remove-vnd.patch (hunk 1 above)
Comment (alarcher) [reply components/openindiana/qemu-kvm/patches/remove-vnd.patch]: Fixed

File: components/openindiana/qemu-kvm/Makefile (hunk 2 above)
Comment (alarcher) [reply components/openindiana/qemu-kvm/Makefile]: Fixed

**Review by AndWac — APPROVED** [review]

**Review by pyhalov — APPROVED** [review]

**Pull request merged**